src/app/internal/consoleapp.cpp

```diff
@@ -211,6 +211,8 @@ void ConsoleApp::applyCommandLineOptions(const CmdOptions& options, IApplication
     notationConfiguration()->setTemplateModeEnabled(options.notation.templateModeEnabled);
     notationConfiguration()->setTestModeEnabled(options.notation.testModeEnabled);
 
+    audioConfiguration()->setAudioDelayCompensate(options.audio.audioDelayCompensate.value_or(1024)); // FIX: equal to buffer-size
```

Comment:
This would need to be moved to `GuiApp`, otherwise it doesn't have any effect, because `ConsoleApp` is only used when running in CLI mode

Reply:
I'll try to move this as a gui preferences option in jack section